src/pykiso/test_coordinator/test_suite.py

```diff
@@ -128,7 +128,7 @@ def __init__(
             wait for a report during teardown execution
         :param test_ids: jama references to get the coverage
             eg: {"Component1": ["Req1", "Req2"], "Component2": ["Req3"]}
-        :param variant: string that allows the user to execute a subset of tests
+        :param tag: string that allows the user to execute a subset of tests
```

Comment:
Same in all of this file regarding the parameter documentation

Reply:
Done